pkg/query/aur_warnings.go

```diff
@@ -21,9 +22,16 @@ func NewWarnings() *AURWarnings {
 }
 
 func (warnings *AURWarnings) Print() {
-	if len(warnings.Missing) > 0 {
+	normalMissing, debugMissing := filterDebugPkgs(warnings.Missing)
+
+	if len(normalMissing) > 0 {
 		text.Warn(gotext.Get("Missing AUR Packages:"))
-		printRange(warnings.Missing)
+		printRange(normalMissing)
+	}
+
+	if len(debugMissing) > 0 {
+		text.Warn(gotext.Get("Missing AUR Debug Packages:"))
```

Comment:
You might have to regenerate the english .po file for this, if you need help with it just ping me and I'll add it after

Reply:
I have looked in the repo to see if there are simple instructions on how to regenerate the .po file, but didn't find a lot.
I'd appreciate some help with that, yes